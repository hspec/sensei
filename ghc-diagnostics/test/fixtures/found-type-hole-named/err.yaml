version: '1.1'
ghcVersion: ghc-9.10.0
span:
  file: test/fixtures/found-type-hole-named/Foo.hs
  start:
    line: 2
    column: 23
  end:
    line: 2
    column: 27
severity: Error
code: 88464
message:
- |-
  Found type wildcard `_foo' standing for `String'
  To use the inferred type, enable PartialTypeSignatures
- 'In the type signature: foo :: FilePath -> IO _foo'
hints: []
