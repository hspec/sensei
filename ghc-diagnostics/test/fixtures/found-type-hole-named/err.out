test/fixtures/found-type-hole-named/Foo.hs:2:23: error: [GHC-88464]
    • Found type wildcard `_foo' standing for `String'
      To use the inferred type, enable PartialTypeSignatures
    • In the type signature: foo :: FilePath -> IO _foo

